client/src/services/stardust/stardustTangleCacheService.ts

```diff
@@ -373,22 +397,25 @@ export class StardustTangleCacheService extends TangleCacheService {
     public async foundryDetails(
         request: IFoundryRequest,
         skipCache: boolean = false
-    ): Promise<IFoundryResponse | undefined> {
-        const cacheEntry = this._stardustSearchCache[request.network][`${request.foundryId}--fdetails`];
+    ): Promise<{ foundryDetails?: IOutputResponse; error?: string }> {
+        const cacheKey = `${request.foundryId}--fdetails`;
```

Comment:
Is cacheKey naming appendix "--fdetails" a typo or intentional?

Reply:
Intentional, just to make the cache keys less likely to collide. Might be that it's not needed for this particular entry, but it can't hurt.